### PR TITLE
Add upgrade test to e2e

### PR DESCRIPTION
**Pull request by dry923**
Add an upgrade test to the e2e scripts. 

Depends-on: https://github.com/cloud-bulldozer/benchmark-wrapper/pull/216
Until ^ is merged it is targeting my repo for testing.

### CODE REVIEW

File: workloads/upgrade-perf/common.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+else
+  echo $cloud_name > uuid.txt
+fi
+
+echo "Installing snafu in python virtual environment"
+
+rm -rf /tmp/snafu upgrade
+
+# REPLACE ONCE WRAPPER IS MERGED
+git clone https://github.com/dry923/snafu.git --branch upgrade /tmp/snafu
```
Comment (mohit-sheth) [inline workloads/upgrade-perf/common.sh:74]: upstream:master before merge

File: workloads/upgrade-perf/common.sh [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+python3 -m venv upgrade
+source upgrade/bin/activate
+pip3 install -e /tmp/snafu
+
+es_index="openshift-upgrade-timings"
+es=${_es}
+es_port=${_es_port}
```
Comment (mohit-sheth) [inline workloads/upgrade-perf/common.sh:82]: where are these variables used?

File: workloads/upgrade-perf/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+### ES_SERVER
+Default: `milton.aws.com`  
+Elasticsearch server to index the results of the current run
+
+### ES_PORT
+Default: ``  
+Port number for elasticsearch server
+
+### COMPARE (not implemented yet)
```
Comment (chaitanyaenr) [inline workloads/upgrade-perf/README.md:47]: We might want to remove the uperf reference.

File: workloads/upgrade-perf/common.sh [hunk 4]
```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+set -x
+
+# Check cluster's health
+if [[ ${CERBERUS_URL} ]]; then
```
Comment (chaitanyaenr) [inline workloads/upgrade-perf/common.sh:5]: We might want to add a Cerberus check at the end of the run as well to make sure the upgraded cluster is healthy. We can query ripsaw as well, for example: https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/network-perf/run_pod_network_test_fromgit.sh#L77. Thoughts?

Comment (chaitanyaenr) [reply workloads/upgrade-perf/common.sh:5]: Think the benchmark will fail if Cerberus reports issues right? In that case we don't need an explicit check.

File: workloads/upgrade-perf/common.sh (hunk 1 above)
Comment (dry923) [reply workloads/upgrade-perf/common.sh:74]: yup. Still waiting on the wrapper merge then i'll change it.

File: workloads/upgrade-perf/common.sh (hunk 2 above)
Comment (dry923) [reply workloads/upgrade-perf/common.sh:82]: snafu expects them to be defined in the environment. Not sure why it cant just accept cmnd line args but thats a different discussion really

File: workloads/upgrade-perf/common.sh (hunk 1 above)
Comment (mohit-sheth) [reply workloads/upgrade-perf/common.sh:74]: ah, missed the description
my bad

File: workloads/upgrade-perf/README.md (hunk 3 above)
Comment (dry923) [reply workloads/upgrade-perf/README.md:47]: HA! Good catch. Also fixed in the in scale doc since its a copy of a copy.

**Comment by mohit-sheth** [comment]
LGTM

**Review by mohit-sheth — APPROVED** [review]

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**